library/core/src/prelude/mod.rs

```diff
@@ -46,12 +65,15 @@ pub mod rust_2021 {
     pub use crate::convert::{TryFrom, TryInto};
 }
 
-/// The 2024 edition of the core prelude.
+/// The 2024 version of the core prelude.
 ///
 /// See the [module-level documentation](self) for more.
 #[unstable(feature = "prelude_2024", issue = "none")]
 pub mod rust_2024 {
-    #[unstable(feature = "prelude_2024", issue = "none")]
+    #[stable(feature = "rust1", since = "1.0.0")]
+    pub use super::common::*;
```

Comment:
What about `FromIterator`? It was in the `rust_2021` prelude, so it should probably also be in the `rust_2024` prelude.

Reply:
Right — I missed that. Likewise with `TryFrom`/`TryInto`.